.github/workflows/test_release_docker_hub.yml

```diff
@@ -0,0 +1,93 @@
+name: Release to Docker Hub
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9]
```

Comment:
Just wondering... Shouldn't these match https://github.com/dashpay/sentinel/blob/master/.travis.yml#L3-L5 ? or should we add these to `.travis.yml` maybe to make sure we don't release smth that passes Travis but fails on Docker release?

Reply:
I removed 2.7 because it is functionally obsolete, and chose 3.8 since this is the version currently shipping in the latest Ubuntu LTS, which we recommend in the docs, and 3.9, which I used for the Docker image in this PR. We are migrating away from Travis, but I'll add these two versions to `travis.yml` for now as well to ensure parity there.